src/types/index.ts

```diff
@@ -26,7 +26,7 @@ export interface ChangeEvent<T = unknown> {
  * second arg with some shortcuts for common use cases.
  */
 export type ChangeHandler<T = unknown, E extends SyntheticEvent = SyntheticEvent> = (
-    event: E,
+    event: E | Event,
```

Comment:
This can be removed now right?

Reply:
nope because a keyboard event still gets passed in

Reply:
Ok, in that case It hink it just needs to move up to the line above, `E extends SyntheticEvent | Event`